pom.xml

```diff
@@ -84,13 +84,15 @@
         <version.ch.qos.cal10n>0.8.1</version.ch.qos.cal10n>
         <version.com.fasterxml.classmate>1.3.3</version.com.fasterxml.classmate>
         <version.com.fasterxml.jackson>2.8.11</version.com.fasterxml.jackson>
+        <version.com.github.ben-manes.caffeine>2.4.0</version.com.github.ben-manes.caffeine>
```

Comment:
This and all the other new dependencies will need adding to wildfly/dist/src/distribution/resources/docs/licenses/licenses.xsl

@psakar and @Skullman can you please let me know if anything else is required?

Reply:
I've added the licenses for the new dependencies to /feature-pack/src/license/full-feature-pack-licenses.xml